projects/components/src/titled-content/titled-content.component.scss

```diff
@@ -11,11 +11,24 @@
     display: flex;
     flex-direction: row;
     align-items: center;
-    justify-items: center;
+
+    &.center {
+      justify-items: center;
+    }
+
+    &.space-between {
+      justify-content: space-between;
```

Comment:
what is the difference between `justify-items` and ` justify-content: ` ?

Reply:
- The justify-items property controls the alignment of grid items. It is set on the grid container. 
- The justify-content property controls the alignment of grid columns. It is set on the grid container. It does not apply to or control the alignment of grid items.

I kept the old property present here with justify-items and I added the new property to the new style

Reply:
okay.. so `justify-items: space-between` would not work here?